NuGet.Config

```diff
@@ -4,6 +4,7 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
+    <add key="coreclr-xunit" value="https://www.myget.org/F/coreclr-xunit/api/v3/index.json" />
```

Comment:
@eerhardt I had to include this feed as a workaround to run desktops tests? Can we get the latest (1.0.0-rc2-157751-46) `dotnet-test-xunit` package to the cli-deps feed?